examples/darwin-framework-tool/templates/tests/ciTests.json

```diff
@@ -66,6 +66,7 @@
         "Disabled due to using provisional Zeolite Filter Monitoring (ZeoliteFilterMonitoring) cluster:",
         "Test_TC_ZEOFREMON_1_1",
         "Test_TC_ZEOFREMON_2_1",
+        "TestOperationalState",
```

Comment:
This most definitly does not use "Zeolite Filter Monitoring".  Please place this in the right place in this list that would have the right reason for it being disabled.